2020-10-13-community-call-dei (#300)
USRSE/usrse.github.io: _posts/2020-10-13-community-call-dei.md

---
layout: post
title: Summary of October 2020 Community Call - US-RSE's Role in Promoting Diversity, Equity and Inclusion
tags: [community call]
posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill
---

> On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.

Reviewer: ```suggestion
On October 8, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.
```

Reviewer: Sorry for not catching this earlier!

> On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.

Reviewer 2: ```suggestion
On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion." The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity, both within the US-RSE organization and externally through outreach to the broader community. 
```
Suggested change. I don't think there was any question about if people supported this, so I removed the last sentence.

Reviewer 3: I agree that the people on the call supported it, but I'm not sure that the whole community does.  Maybe this would be better as
> The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession through outreach to the broader community, and to determine if the participants in the call supported the idea of taking such steps.

> We initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.

Reviewer 2: The post opens talking about the group in the third person, so it's strange to see "we" here - can we choose one point of view and be consistent?

Reviewer 3: Maybe we change the 2nd paragraph to:
> The call attendees initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.

The problem is there are 2 potential "we"s here -the people on the call and the US-RSE.

Author: Perhaps replace "so in essence, we say" with "intimates" or "suggests"
> The call attendees initially talked about the fact that US-RSE's Code of Conduct includes some discussion of diversity, but that our mission does not currently include diversity, which intimates that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.

Reviewer 3: I don't think that addresses the problem

Reviewer: I agree it is a bit confusing. There are a lot of we's and I don't mind it (sounds more personal and approachable) What about adding a section header after the first paragraph? So then it has a logical break and transition from the 3rd person in the first paragraph.

```suggestion
### Call Overview & Synopsis

We initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.
```

It was suggested that members take a look at the [slides](https://doi.org/10.6084/m9.figshare.12955094.v4) from a talk given by Neil Chue Hong, Director of the Sustainability Software Institute. Neil talked about the lack of diversity within research software engineering, even when compared to academia as a whole or computer science. This led to a rich discussion about possible future directions and initiatives to improve diversity and inclusion:

-   Adding a page to our website with resources for members who want to learn about diversity and inclusion in general, as well as practices they can use in their own lives. An example of such resources is guidelines on how to write good references for students while avoiding gender bias.

> -   Making sure we are all inclusive in the hiring process without giving applicants the perception that they are only invited because they belong to a minority, or perpetrating a "diversity hire". Broadening the pool of interested applicants through education about the field might prevent this, which highlights the importance of outreach outside our immediate community.

Reviewer 2: ```suggestion
-   Making sure we are all inclusive in the hiring process without giving applicants the perception that they are only invited because they belong to a minority, or perpetrating a "diversity hire." Broadening the pool of interested applicants through education about the field might prevent this, which highlights the importance of outreach outside our immediate community.
```

Reviewer 3: I agree with this change

-   Outreach about the RSE profession to on-campus groups with student participation. It was noted that since events are virtual these days, that now might be a good time to try this form of outreach.

We also discussed the RSE Stories podcast as a current success for community building and increasing the representation of diverse voices, and the potential to expand into topics of diversity and inclusion. It was brought up that groups use social media platforms differently, and that outreach should include written and spoken communication with disability access in mind. A new channel on the US-RSE slack, [\#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP), was created to continue further discussion. This channel could be a place for also sharing individual activities that others could get involved in. Numerous participants on the call mentioned activities that they were personally involved in that were of interest to other members.

> ![Community Call about Diversity and Inclusion](/assets/posts/2020-10-13-community-call-dei.png)

Reviewer 3: ```suggestion
![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png)
```

Reviewer 3: Actually, I think it should be
![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png "Community Call about Diversity and Inclusion")

> ![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png)

Reviewer 3: ```suggestion
![Community Call about Diversity and Inclusion]({{ site.baseurl }}/assets/posts/2020-10-13-community-call-dei.png "Community Call about Diversity and Inclusion")
```

---
layout: post
title: Summary of October 2020 Community Call - US-RSE's Role in Promoting Diversity, Equity and Inclusion
tags: [community call]

> posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill

Reviewer 3: ```suggestion
posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill, Vanessa Sochat
```

Reviewer 2: oops I did the same thing, this one looks good!

---

> On October 8, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.

Reviewer 3: ```suggestion
On October 8, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession through outreach to the broader community, and to determine if the participants in the call supported the idea of taking such steps.
```

> On October 8, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession through outreach to the broader community, and to determine if the participants in the call supported the idea of taking such steps.

Reviewer 2: ```suggestion
On October 8, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion." The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity within the US-RSE organization and externally in our profession through outreach to the broader community, and to determine if the participants in the call supported the idea of taking such steps.
```
commas, periods, etc. always go inside quotation marks

> posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill

Reviewer 2: ```suggestion
posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill, Vanessa Sochat
```
Offered by @danielskatz !
